_component_design/buttons.md

```diff
@@ -14,203 +14,194 @@ last-modified: 2017-08-17
 
 <div class="hxRow">
-{% column left:"hxCol" %}
--   Use the primary action button to prompt the user to take action.
 
--   In most instances, use the standard size button. Use the compact buttons
-    with space constraints such as within a table.
 
--   Large buttons should be used in mobile designs to accommodate the ideal
-    touch target size.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+Use a button to launch a process or initiate an action. Do not use buttons for navigation.  When the user needs to get from one page to another, use a text link. 
+
+### Best practice
+{: .hxSubSectionTitle}
+
+- Button Text style should always be Title Case where the first letter of each word is capitalized.
+- Buttons should always use the labelling convention of “Noun Verb”, with no articles. e.g. “Create Server”, not “Create a Server”.
+- Keep your text labels as short as possible while also fully communicating what action the button will trigger.
+- When choosing which button weight to use, you will probably use a secondary button unless your button need meets the requirements of a primary button, or tertiary button.
+- When choosing which button size to use consider the context. In most cases, you will use a standard button
+- There should only be one primary button in a form.
+- The primary button should always represent an inherently apparent next step. e.g.  "Create Server", "Add Selected", "Confirm Deletion",  "Submit”, “Next”.
+- If you have multiple buttons on a page, there can only be one primary button.
 
--   Use one primary button per page.
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Primary button sizes"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-primary-sizes.svg){:width="100%"}
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Primary button states"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-primary-actions.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the button pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-hero.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
-</div>
 
-# Secondary Buttons
+</div>
 
+## Simple buttons
+{: .hxSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
--   The secondary button can be used in conjunction with the primary button or
-    when there are multiple buttons on the same screen.
 
--   The same sizing rules apply to both secondary and primary buttons.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+When using a simple button you will need to specify both a weight and size.
```

Comment:
Try to avoid "need" in technical docs.
"When using a simple button, specify both weight and size."

Reply:
updated per suggestion.